lawnchair/res/values/config.xml

```diff
@@ -104,6 +104,7 @@
     <bool name="config_default_show_icon_labels_in_drawer">true</bool>
     <bool name="config_default_enable_fuzzy_search">false</bool>
     <bool name="config_default_enable_smartspace">true</bool>
+    <bool name="config_default_separate_recent_activity">false</bool>
```

Comment:
Do we still need this? I think this can be safely removed.

Reply:
yeah let me remove it